pkgs/os-specific/linux/facetimehd/default.nix

```
@@ -17,8 +17,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "patjak";
     repo = "bcwc_pcie";
```

I think the repository was renamed
```suggestion
    repo = "facetimehd";
```

---

A long time ago, just been lazy and relying on the redirect ;)